source/_integrations/tasmota.markdown

```diff
@@ -24,13 +25,16 @@ This integration allows you to control [Tasmota](https://tasmota.github.io/docs/
 
 ## Limitations
```

Comment:
Maybe this shouldn't be called limitations anymore, perhaps "Supported Features" would be be better?

Reply:
Right. Since we are almost in par with the MQTT integration (just shutters ar missing) is time to change it.